include/fmt/chrono.h

```diff
@@ -563,7 +564,13 @@ template <typename Char> struct formatter<std::tm, Char> {
     while (end != ctx.end() && *end != '}') ++end;
     auto size = detail::to_unsigned(end - it);
     specs = {it, size};
-    if (specs == string_view("%F", 2)) spec_ = spec::year_month_day;
+    // basic_string_view<>::compare isn't constexpr before C++17
```

Comment:
There is no `basic_string_view<>` before C++17.

Could it be that the internal replacement should be updated?

Reply:
Problem in ```std::char_traits<>::compare```:
https://github.com/fmtlib/fmt/blob/3d0c7ae385bcff3d0c152c358db67a222dd0628c/include/fmt/core.h#L472-L478

constexpr ```std::char_traits<>::compare``` starts from C++17 only